src/encoded/types/user.py

```diff
@@ -80,7 +80,11 @@ def __ac_local_roles__(self):
         },
     }, category='page')
     def access_keys(self, request):
-        if not request.has_permission('view_details'):
+        properties = self.properties
+        is_admin = 'groups' in properties and 'admin' in properties['groups'] 
```

Comment:
Lines 84 - 87 do not seem right. But I could not think of a better way.

Reply:
This again should be assigned in ACLs. And in any case this is already `view_details` is already restricted to admin group so this is unnecessary.

Reply:
Should maybe this be checked in access_key.AccessKey.access_key_add()